records/generator_perf_test.go

```diff
@@ -0,0 +1,37 @@
+package records
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+)
+
+func BenchmarkInsertRR(b *testing.B) {
```

Comment:
the func name is pretty clear, and corresponds to golang test func naming standards - it's testing the `insertRR` func